lib/src/modules/task.cc

```diff
@@ -108,6 +123,15 @@ Task::Task(const fs::path& exec_prefix,
 
     input_validator_.registerSchema(input_schema);
     results_validator_.registerSchema(output_schema);
+
+    // TODO: Remove this. Only put it here to test out the SSL stuff.
+    // ca_ = "/Users/enis.inan/GitHub/orchestrator/dev-resources/ssl/ca.pem";
```

Comment:
Didn't think of it, the above was just quick and easy at the time. Thanks for the suggestion!